apps/balancer-tools/src/app/apr/round/(components)/PoolListTable.tsx

```diff
@@ -178,21 +198,53 @@ function TableRow({
   const router = useRouter();
   return (
     <Table.BodyRow
-      classNames="hover:bg-blue4 hover:cursor-pointer"
+      classNames="hover:bg-blue4 hover:cursor-pointer duration-500"
```

Comment:
what are we changing the duration of here?

Reply:
To make it more similar to Balancer's visual